src/bin/wasm-tools/validate.rs

```diff
@@ -97,31 +97,31 @@ impl Opts {
 fn parse_features(arg: &str) -> Result<WasmFeatures> {
     let mut ret = WasmFeatures::default();
 
-    const FEATURES: &[(&str, fn(&mut WasmFeatures) -> &mut bool)] = &[
-        ("reference-types", |f| &mut f.reference_types),
-        ("function-references", |f| &mut f.function_references),
-        ("simd", |f| &mut f.simd),
-        ("threads", |f| &mut f.threads),
-        ("shared-everything-threads", |f| {
-            &mut f.shared_everything_threads
+    const FEATURES: &[(&str, fn(&mut WasmFeatures, new_value: bool))] = &[
```

Comment:
If you're up for it this might actuall be best modeled as `&[(&str, WasmFeatures)]` instead of a bunch of closures and that might help clean this up a bit

Reply:
Technically we could maybe even make use of this method if all names match:
https://docs.rs/bitflags/2.5.0/bitflags/trait.Flags.html#method.from_name